Source/MetroXHunter/Sequencer/LevelSequencerHandler.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Implemented by Corentin Paya
+ */
+
+#include "Sequencer/LevelSequencerHandler.h"
+#include "LevelSequenceActor.h"
+#include "CineCameraActor.h"
+
+ALevelSequencerHandler::ALevelSequencerHandler()
+{
+	PrimaryActorTick.bCanEverTick = false;
+
+	SceneRoot = CreateDefaultSubobject<USceneComponent>( TEXT( "Scene Root" ) );
+	RootComponent = SceneRoot;
+
+	// Define the spawn parameters
+	FActorSpawnParameters SpawnParams;
+	SpawnParams.Owner = this;
+
+	UWorld* World = GetWorld();
+	if ( !World )
+		return;
```

Comment:
Remember, no return on the next line
![](https://static.wikia.nocookie.net/nbk/images/d/de/No_russian_mw2.jpg/revision/latest?cb=20101022203742)